src/test/common/steps/cookie-steps.js

```diff
@@ -1,9 +1,17 @@
-const { Then } = require('cucumber')
-const { expect } = require('chai')
+const { When, Then } = require('cucumber')
+const { assert, expect } = require('chai')
 
 const pages = require('./pages')
 
-const GOV_UK_TABE_CLASSNAME = 'govuk-table'
+const GOV_UK_TABLE_CLASSNAME = 'govuk-table'
+
+When(/^I go directly to the cookies page$/, async function () {
```

Comment:
You don't need this, use `I navigate to the cookies page` which is in `navigation.js`.

Reply:
Removed.